### ISSUE TITLE
Should we upgrade jQuery to 3.4.1 because of known security vulnerabilities?

### ISSUE THREAD

**Issue by asdf2014** [theme-next/hexo-theme-next#886]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

https://yuzhouwan.com without known security vulnerabilities

### Actual behavior 

![image](https://user-images.githubusercontent.com/8108788/58367393-3e6ccc80-7f11-11e9-845e-b061d9c8bf62.png)

### Steps to reproduce the behavior 
1. Open https://yuzhouwan.com
2. Run lighthouse

### Node.js and NPM Information

```
v11.6.0
6.9.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.8.0"
  },
  "dependencies": {
    "@babel/core": "^7.4.5",
    "browser-sync": "^2.26.5",
    "gulp": "^4.0.2",
    "gulp-htmlclean": "^2.7.22",
    "gulp-htmlmin": "^5.0.1",
    "gulp-imagemin": "^5.0.3",
    "gulp-install": "^1.1.0",
    "gulp-minify-css": "^1.2.4",
    "gulp-uglify": "^3.0.2",
    "hexo": "^3.8.0",
    "hexo-abbrlink": "^2.0.5",
    "hexo-autonofollow": "^1.0.1",
    "hexo-deployer-git": "^1.0.0",
    "hexo-filter-flowchart": "^1.0.4",
    "hexo-filter-mermaid-diagrams": "^1.0.5",
    "hexo-fs": "^1.0.2",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-feed": "^1.2.2",
    "hexo-generator-index": "^0.2.1",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-sitemap": "^1.1.2",
    "hexo-generator-tag": "^0.2.0",
    "hexo-inject": "^1.0.0",
    "hexo-related-popular-posts": "^3.0.5",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-kramed": "^0.1.4",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.3",
    "hexo-tag-mermaid": "^1.0.0",
    "mermaid": "^8.0.0",
    "natives": "^1.1.6",
    "phantomjs": "^2.1.7"
  },
  "devDependencies": {
    "babel-core": "^6.26.3",
    "babel-preset-env": "^1.7.0",
    "hexo-util": "^0.6.3"
  }
}
```

#### Hexo version

```
hexo: 3.8.0
hexo-cli: 1.1.0
os: Darwin 18.5.0 darwin x64
node: 11.6.0
v8: 7.0.276.38-node.13
uv: 1.24.1
zlib: 1.2.11
ares: 1.15.0
modules: 67
nghttp2: 1.34.0
napi: 3
llhttp: 1.0.1
http_parser: 2.8.0
openssl: 1.1.0j
icu: 63.1
unicode: 11.0
cldr: 34.0
tz: 2018e
```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [x] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

**Comment by 1v9** [comment]
I agree to upgrade the internal jQuery to [v3](https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js)  because I have been using it for a long time and it works well.

**Comment by ivan-nginx** [comment]
@1v9 if u sure, I think u can do it.

**Comment by stevenjoezhang** [comment]
Fixed in #903

**Comment by stevenjoezhang** [comment]
jQuery dropped in v7.4.0 ~
